packages/components/src/navigation/stories/default.js

```diff
@@ -26,6 +26,14 @@ export function DefaultStory() {
 					onClick={ () => setActiveItem( 'item-2' ) }
 					title="Item 2"
 				/>
+				<NavigationItem>
+					<a
+						className="components-button"
+						href="http://www.example.com"
+					>
+						External Link
+					</a>
```

Comment:
Would it make sense to import an actual `Button` component, and making it render as `a` with a `href` prop?
E.g.
```jsx
import Button from '../button';

// ...

<NavigationItem>
  <Button href="http://www.example.com">External Linx</Button>
</NavigationItem>
```

Reply:
We could, but I figured `<a>` markup would get the point across better than `<Button>` because you'd need knowledge about that component to understand what was happening under the hood.